packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -502,16 +503,59 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     return encodeAbiParameters(parseAbiParameters("bytes, address"), [moduleSignature, moduleAddressToUse]);
   }
 
+  async #getPaymasterUserOp(
```

Comment:
What's the point of "#" in front of these ? For "internal" functions ?

Reply:
Yeah they denote private class properties

Reply:
Can we stick do the current implementation of using "private" for private functions ? Or is there any reason using # would be better ?